pkgs/ffigen/test/native_objc_test/deprecated_test.m

```diff
@@ -0,0 +1,94 @@
+#import <Foundation/NSObject.h>
+
```

Comment:
I also see the concept "obsolete" in your code. Should we have an obsolete input as well in this file?